doc/languages-frameworks/python.section.md

```diff
@@ -2017,10 +2017,12 @@ The following rules are desired to be respected:
 * `meta.platforms` takes the default value in many cases.
   It does not need to be set explicitly unless the package requires a specific platform.
 * The file is formatted with `nixfmt-rfc-style`.
-* Commit names of Python libraries should reflect that they are Python
-  libraries, so write for example `python311Packages.numpy: 1.11 -> 1.12`.
-  It is highly recommended to specify the current default version to enable
-  automatic build by ofborg.
+* Commit names of Python libraries must reflect that they are Python
+  libraries (e.g. `python311Packages.numpy: 1.11 -> 1.12`).
+* The current default version of python should be included
+  in commit messages to enable automatic builds by ofborg.
+  For example `python311Packages.numpy: 1.11 -> 1.12` should be used rather
+  than `python3Packages.numpy: 1.11 -> 1.12`.
```

Comment:
IIRC, this is out of date and ofborg will build both package sets if you use `python3Packages`. Therefore this advice is actually counterproductive.

Disclaimer: I have not verified this recently.

Reply:
I *believe* that https://github.com/NixOS/nixpkgs/pull/165994 fixed this, but maybe it regressed since or something.

Reply:
python3Packages does not work on ofborg, I (wearily) nit about it for that reason.

Reply:
Also just to encourage good practices from people who are reading too quickly, we should either specify python312Packages specifically or say python3xxPackages to be more general IMO.

Reply:
I swear it worked at one point. Maybe I’m just imagining things or maybe it broke again. I’m not the only person who remembered it working since someone else said to me recently that it had been fixed at some point. :(

Reply:
I have been checking python libraries for over a year and have never had it work. Maybe they missed that someone ran it manually.
People don't read much documentation, but they often ask reviewers to show them documentation.
It needs to be clearly stated to avoid unnecessary discussion.

Reply:
https://github.com/NixOS/ofborg/issues/577

Reply:
I think `python3{11,12}Packages.…` should work, right? Maybe we should prefer that to check both supported versions.

Reply:
Personally, I don't want to complicate it because that can be checked by nixpkgs-review.
In my experience, just checking one version is usually enough to catch build failures on aarch64-linux or darwin that are often overlooked.